Cloud changelog - April 13th (#6307)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2023-04-13](#release-2023-04-13)
- [Release 2023-03-29](#release-2023-03-29)
- [Release 2023-03-20](#release-2023-03-20)
- [Release 2023-01-26](#release-2023-01-26)
- [Release 2023-01-16](#release-2023-01-16)
- [Release 2022-12-20](#release-2022-12-20)

## Release 2023-04-13

### Improvements

#### User Interface (UI)
 - Added a marketplace button to the bottom of the apps bar. The button is visible if the marketplace is enabled and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
 - Added a "Add channels" button to the bottom of the left-hand sidebar to make it more obvious for users on how to create or join channels.

#### Administration

>  - The following repositories have been merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. Playbooks and Boards are now core parts of the product that can't be disabled.

Reviewer: ```suggestion
 - The following repositories have been merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. Playbooks and Boards are now core parts of the product that cannot be disabled.
```

Latest Mattermost Cloud releases:

- [Release 2023-04-13](#release-2023-04-13)
- [Release 2023-03-29](#release-2023-03-29)
- [Release 2023-03-20](#release-2023-03-20)
- [Release 2023-01-26](#release-2023-01-26)
- [Release 2023-01-16](#release-2023-01-16)
- [Release 2022-12-20](#release-2022-12-20)

## Release 2023-04-13

### Improvements

#### User Interface (UI)

>  - Enabled Work Templates in Onboarding checklist and `+` Menu.

Reviewer 2: ```suggestion
 - Enabled Work Templates in the Onboarding checklist and the `+` Menu.
```

>  - Added a marketplace button to the bottom of the apps bar. The button is visible if the marketplace is enabled and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.

Reviewer 2: ```suggestion
 - Added a **Mattermost Marketplace** option to the bottom of the apps bar. The option is visible when the Marketplace is enabled, and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
```

>  - Added a "Add channels" button to the bottom of the left-hand sidebar to make it more obvious for users on how to create or join channels.

Reviewer 2: ```suggestion
 - Added an **Add channels** button to the bottom of the left-hand sidebar to make the action more obvious for users who want to create or join channels.
```

>  - Removed Webapp Build Hash since it is now identical to Server Build Hash.

Reviewer 2: Can we specify where the Webapp Build Hash details have been removed please? (I believe it's the About modal, correct?)

#### Administration
 - The following repositories have been merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. Playbooks and Boards are now core parts of the product that cannot be disabled.

>  - The file info stats query is now optimized by denormalizing the ``channelID`` column into the table itself. This will speed up the query to get the file count for a channel on clicking the right-hand side. Migration times:

Reviewer 2: ```suggestion
 - The file info stats query is now optimized by denormalizing the ``channelID`` column into the table itself. This will speed up the query to get the file count for a channel when selecting the right-hand pane. Migration times:
```

Reviewer 2: Engineering: Please confirm that my editorial suggestions haven't changed the intended meaning of this bullet point. Thanks!

   - On a MySQL 8.0.31 DB with 1405 rows in FileInfo and 11M posts, it took around 0.3s

>    - On a PostgreSQL 12.14 DB with

Reviewer 2: Can the second and third bullet points be combined into a single bullet point, please?

   - 1731 rows in FileInfo and 11M posts, it took around 0.27s

>  - Added the ability to search by part of the first name, last name, nickname or username in **Sytem Console > Users** section.

Reviewer 2: ```suggestion
 - Added the ability to search a partial first name, last name, nickname, or username on the **System Console > Users** page.
```

 - **Contact Support** now redirects users to Zendesk and pre-fills known information.

### Bug Fixes
 - Fixed a scrolling issue in the purchase modals.
 - Fixed an issue where the **Delete Category Dialog** message was not visible in Boards.
 - Fixed an issue where the experimental Shared Channels feature failed to synchronize if a previously removed table column was still present.

### Known Issues
 - The URL of the post in a reminder post for Direct and Group Messages have a double slash on mobile [MM-51026](https://mattermost.atlassian.net/browse/MM-51026).

>  - A user gets scrolled to the bottom of the post editor after pasting long text in the right-hand side [MM-51302](https://mattermost.atlassian.net/browse/MM-51302).

Reviewer 2: ```suggestion
 - A user gets scrolled to the bottom of the post editor after pasting long text in the right-hand pane [MM-51302](https://mattermost.atlassian.net/browse/MM-51302).
```

   - On a MySQL 8.0.31 DB with 1405 rows in FileInfo and 11M posts, it took around 0.3s
   - On a PostgreSQL 12.14 DB with 1731 rows in FileInfo and 11M posts, it took around 0.27s

 - Added the ability to search a partial first name, last name, nickname, or username on the **System Console > Users** page.
 - **Contact Support** now redirects users to Zendesk and pre-fills known information.
 - Added a mechanism for public routes on products and used it to support publicly shared Board links.

### Bug Fixes
 - Fixed a scrolling issue in the purchase modals.
 - Fixed an issue where the **Delete Category Dialog** message was not visible in Boards.
 - Fixed an issue where the experimental Shared Channels feature failed to synchronize if a previously removed table column was still present.
 - Fixed an innocuous panic in Boards Rest API when requesting files and an error other than ``not found`` is encountered.
 - Fixed an issue with the compact message mode.

### Known Issues

>  - The URL of the post in a reminder post for Direct and Group Messages have a double slash on mobile [MM-51026](https://mattermost.atlassian.net/browse/MM-51026).

Reviewer 3: ```suggestion
 - The URL of the post in a reminder post for direct and group messages have a double slash on mobile [MM-51026](https://mattermost.atlassian.net/browse/MM-51026).
```
